stdlib/public/Synchronization/Atomics/WordPair.swift

```diff
@@ -183,6 +183,18 @@ extension WordPair: Hashable {
   }
 }
 
+@available(SwiftStdlib 6.2, *)
```

Comment:
Should this pull request be cherry-picked?

```suggestion
@available(SwiftStdlib 6.1, *)
```